core/src/main/java/org/mule/runtime/core/internal/util/JdkVersionUtils.java

```diff
@@ -295,6 +296,8 @@ public static void validateJdk() throws RuntimeException {
     if (!isSupportedJdkVendor()) {
       logger.info("You're executing with a JDK made by a vendor that is not on the recommended list of vendors. Vendor: "
           + JAVA_VENDOR + " Please consider changing to a recommended JDK vendor.");
+    } else if (getBoolean(getProperty("mule.jdkDebug"))) {
```

Comment:
make the property into a constant, use the MULE prefix from the `MuleProperties`